src/components/SortBar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import FilterDropDown from './FilterDropdown';
+
+const sort = {
+  amountLow: 'Amount (Low to High)',
+  amountHigh: 'Amount (High to Low)',
+  deadlineEarly: 'Deadline (Earliest to Latest)',
+  deadlineLatest: 'Deadline (Latest to Earliest)',
+};
+
+function FilterBar() {
```

Comment:
This doesn't match the file name

Reply:
This file no longer exist since it got merged in the FilterBar class